src/libs/DistanceRequestUtils.js

```diff
@@ -90,7 +90,8 @@ const getDistanceMerchant = (hasRoute, distanceInMeters, unit, rate, currency, t
     const singularDistanceUnit = unit === CONST.CUSTOM_UNITS.DISTANCE_UNIT_MILES ? translate('common.mile') : translate('common.kilometer');
     const unitString = distanceInUnits === 1 ? singularDistanceUnit : distanceUnit;
     const ratePerUnit = rate ? PolicyUtils.getUnitRateValue({rate}, toLocaleDigit) : translate('common.tbd');
-    const currencySymbol = CurrencyUtils.getCurrencySymbol(currency) || `${currency} `;
+    // Set the currency symbol only if rate is valid
```

Comment:
This seems unnecessary please remove this

Reply:
Sure. The unnecessary comment is removed now.